### DISCUSSION TITLE
2.4 inch TFT LCD screen SPI serial port 10PIN resolution 240x320 ST7789V with ESP32

### DISCUSSION THREAD

**Discussion by shamirtahir50** [Bodmer/TFT_eSPI#2504] [Compatible displays and setup files]
Hi @Bodmer,

I am trying a chinese display. It has following pinout,

![image](https://user-images.githubusercontent.com/66121435/225597752-a240da03-5ce2-48eb-a09b-c8dae9f303ef.png)

I am confused about the SDA, SD0 and LEDK.

Is there any existing configuration or I need to manually uncomment lines.

**Comment by Bodmer** [comment]
SDA=MOSI
SDO=MISO
LEDK=LED  cathode so connec to GND

Example connections in :Setup42_ILI9341_ESP32.h

Change driver to in that file to
#define  ST7789_DRIVER
Edit User_Setup_Select.h to load that header file.

**Reply by shamirtahir50** [reply]
@Bodmer Thanks. The upper configuration works fine.

I want to adjust brightness of my screen. Does library supports this? Is there any specific function?

**Comment by Tomblarom** [comment]
Hey I'm actually trying to accomplish something similar. I wanted to hook up a [generic display (without PCB) and only flex cable](https://aliexpress.com/item/4000019186713.html). I know that there are some components on the PCBs, but are they necessary to interface via SPI? Can this be looked up somewhere? I thought about using an FPC-10pin breakoutboard and connect it to my ESP32-C3 this way..

![image](https://github.com/Bodmer/TFT_eSPI/assets/12703270/2ca7ef78-a7c2-45c1-82a4-af52ee9a774c)


Any input?
